src/components/Misc/RsvProgressBar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  current: string;
+};
+
+const states = {
+  idea: 'Idee',
+  agreement_process: 'Prüfung',
+  planning: 'Entwurf',
+  in_progress: 'Umsetzung',
+  done: 'Abgeschlossen',
+};
+
+const baseStyle =
+  'inline-flex items-center justify-center py-3 px-3.5 lg:px-2.5 text-xs lg:text-xs sm:text-base xl:py-2.5 xl:px-4 font-bold mr-1 xl:text-lg hover:cursor-default';
+const selectedStyle = 'bg-slate-900 text-white';
+const unselectedStyle = 'bg-gray-300 text-gray-600';
+
+export const RsvProgressBar: React.FC<Props> = ({ current }) => {
+  const stateList = Object.keys(states);
+  const nStates = stateList.length;
+  return (
+    <div className="flex flex-row">
+      {stateList.map((state, i) => {
```

Comment:
Ein bisschen Geschmackssache, aber ich mache zZ immer häufiger `Object.entries` statt `Object.keys`. Dann kann man sagen:

```suggestion
      {Object.entries(states).map(([key, title], i) => {
```

… und dann direkt auf `title` zugreifen.

In dem Case hier, bevorzuge ich einen Value der klarer sagt, dass es ein Key ist, den wir hier haben, bspw:

```suggestion
      {stateList.map((stateKey, i) => {
```

Reply:
jo finde ich auch so schicker!